discovery/utils/utils.py

```diff
@@ -260,7 +262,8 @@ class FileUtils:
     def __read_service_configuration_file(file_name):
 
         # Check if config file exists. We return an empty dictionary if there isn't any config
-        file_path = f"service/config/{file_name}"
+        parent_dir = dirname(dirname(realpath(__file__)))
```

Comment:
Did you try with setting up PYTHONPATH as per readme file?

Reply:
I ran these commands from the inventory